feat: Update home page (#3289)
kubesphere/website: content/en/_index.md

---
title: KubeSphere Container Platform-The Kubernetes platform tailored for hybrid multicloud
topic: KubeSphere <br/>Community Edition
description: KubeSphere is a distributed operating system for cloud-native application management, using Kubernetes as its kernel. It provides a plug-and-play architecture, allowing third-party applications to be seamlessly integrated into its ecosystem.

css: scss/index.scss

section1:
  title: KubeSphere Community Edition
  topic: KubeSphere Container Platform-The Kubernetes platform tailored for hybrid multicloud
  content: KubeSphere is a distributed operating system for cloud-native application management, using Kubernetes as its kernel. It provides a plug-and-play architecture, allowing third-party applications to be seamlessly integrated into its ecosystem.
  btns:

>     - content: Experience Now

Reviewer: ```suggestion
    - content: Try Free
```

  children:
    - name: KubeKey
      icon: /images/home/kube-key.svg
      content: Rapid installation and operation tools to accelerate Kubernetes deployment, adapt to multiple environments, and automate full lifecycle management.

    - name: OpenELB
      icon: /images/home/open-elb.svg
      content: Native load balancing solution that fills the gap for bare-metal LB, supports multiple network modes, and eliminates external dependencies.

    - name: KubeEye
      icon: /images/home/kube-eye.svg
      content: Kubernetes cluster detection and diagnostic tools to identify configuration risks and compliance issues, diagnose cluster anomalies, and enhance Kubernetes stability and security.

    - name: Fluent Operator
      icon: /images/home/fluent-operator.svg
      content: Cloud-native log and data collection engine that unifies log collection and processing, extends Fluentd / Fluent Bit, enhances operational observability, and has been donated.

    - name: OpenFunction
      icon: /images/home/open-function.svg

>       content: Cloud-native, open-source FaaS (Function as a Service) framework focusing on business logic development, abstracting the underlying runtime environment, and accelerating the implementation of cloud-native functions.

Reviewer: ```suggestion
      content: Cloud-native, open-source FaaS (Function as a Service) framework that focuses on business logic development, abstracts underlying runtime environments to accelerate cloud-native FaaS adoption.
```
